samples/Uno.Toolkit.Samples/Uno.Toolkit.Samples.Shared/Content/Controls/FlipViewSamplePage.xaml

```diff
@@ -0,0 +1,95 @@
+﻿<Page
+	x:Class="Uno.Toolkit.Samples.Content.Controls.FlipViewSamplePage"
+	xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+	xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+	xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+	xmlns:local="using:Uno.Toolkit.Samples.Content.Controls"
+	xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+	xmlns:msui="using:Microsoft.UI.Xaml.Controls"
+	xmlns:sample="using:Uno.Toolkit.Samples"
+	xmlns:utu="using:Uno.Toolkit.UI"
+	Background="{ThemeResource ApplicationPageBackgroundThemeBrush}"
+	mc:Ignorable="d">
+
+	<sample:SamplePageLayout IsDesignAgnostic="True">
+		<sample:SamplePageLayout.DesignAgnosticTemplate>
+			<DataTemplate>
+				<StackPanel Margin="0,20" Spacing="20">
+
+					<StackPanel.Resources>
+						<ResourceDictionary>
+							<Style TargetType="x:TextBlock">
+								<Setter Property="HorizontalAlignment" Value="Center" />
+							</Style>
+						</ResourceDictionary>
+					</StackPanel.Resources>
+
+					<TextBlock Text="Try to navigate throw the FlipView using the buttons inside it. " />
```

Comment:
Consider adding `TextWrap="Wrap"` to text-blocks that might be too long. So, the sentence don't get clipped.